Mathlib/MeasureTheory/Function/StronglyMeasurable/Basic.lean

```diff
@@ -338,6 +338,22 @@ protected theorem mono {m m' : MeasurableSpace α} [TopologicalSpace β]
       (SimpleFunc.finite_range (hf.approx n))
   exact ⟨f_approx, hf.tendsto_approx⟩
 
+protected theorem fst {m : MeasurableSpace α} [TopologicalSpace β] [TopologicalSpace γ]
+    {f : α → β × γ} (hf : StronglyMeasurable f) : StronglyMeasurable fun x ↦ (f x).1 := by
```

Comment:
Is it just `continuous_fst.comp_stronglyMeasurable hf`? Similarly for other new lemmas.